checked_yaml/test/example_test.dart

```diff
@@ -2,171 +2,111 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'dart:convert';
-import 'dart:io';
-
-import 'package:path/path.dart' as p;
+import 'package:checked_yaml/checked_yaml.dart';
 import 'package:test/test.dart';
-import 'package:test_descriptor/test_descriptor.dart' as d;
-import 'package:test_process/test_process.dart';
 
-void main() {
-  test('valid input', () async {
-    final proc = await _run('{"name": "bob", "count": 42}');
+import '../example/example.dart' as example;
```

Comment:
This is pretty naughty, you won't ever be able to use `build_vm_compilers` if you do this, but :man_shrugging:

Reply:
agreed 🤷‍♂

Reply:
@jakemac53 to file a bug!